db_test.go

```diff
@@ -1117,6 +1118,93 @@ func TestTypeAlias_inMappedStruct(t *testing.T) {
 	}
 }
 
+const versionedUsersDDL = `
+CREATE TABLE users (
+  id   BIGINT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  ver  INT NOT NULL
```

Comment:
In typical usage, would you expect implementations to use a default value for the version?